### DISCUSSION TITLE
Firebase Token Error IOS

### DISCUSSION THREAD

**Discussion by fatemafreezewala** [invertase/react-native-firebase#4973] [Q&A] [answered]
Please Solve This 
 Possible Unhandled Promise Rejection (id: 1):
TypeError: _messaging.default.getToken is not a function. (In '_messaging.default.getToken()', '_messaging.default.getToken' is undefined)

**Comment by andersonaddo** [comment, marked as answer]
This should be a github issue, and when you create it follow the template
